app/models/email.rb

```diff
@@ -0,0 +1,19 @@
+class Email
+  include ActiveModel::Validations
+  include ActiveModel::Conversion
+  extend ActiveModel::Naming
+
+  attr_accessor :hide_recipients, :recipients, :reply_to, :subject, :content
+
+  validates_presence_of :hide_recipients, :recipients, :reply_to, :subject, :content
+
+  def initialize(attributes = {})
```

Comment:
general attributes, it implements the general interface for Email.new(params)
of cause in this case it only accepts email specific attributes, but the ActiveModel interface is general